_docs_integrate/delete-an-attribute.md

```diff
@@ -0,0 +1,135 @@
+  - implementation status: OPEN
+  - documentation status: OPEN
+  - published:
+  - link: integrate/delete-an-attribute
+require:
+required_by:
+---
+
+The exact process of deleting an Attribute depends on the kind of Attribute at hand.
+[Creating an Attribute]({% link _docs_integrate/create-attribute-for-yourself.md %}), we must distinguish between [IdentityAttributes]({% link _docs_integrate/data-model-overview.md %}#identityattribute) and [RelationshipAttributes]({% link _docs_integrate/data-model-overview.md %}#relationshipattribute).
+In the former case, a so-called RepositoryAttribute is created, which is a [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute) without `shareInfo`, that you are the `owner` of.
+Afterwards, you may [share it]({% link _docs_integrate/share-own-attribute-to-peer.md %}) with a peer, which yields the creation of an own shared IdentityAttribute.
+This is a LocalAttribute with the copied `content` of the RepositoryAttribute, but an additional `shareInfo` property.
+Doing so, also a LocalAttribute with the same `content` and a respective `shareInfo` is created for the peer, which is referred to as peer shared Attribute.
+In the case of RelationshipAttributes we have own shared and peer shared RelationshipAttributes analogeously, however, no unshared LocalAttributes like we have in the case of IdentityAttributes with RepositoryAttributes.
+These different kinds of Attributes have different demands that need to be taken into account, wanting to delete them.
+
+In general, you can only delete data from your own wallet and never from the peer's.
+So, for example you can delete Attributes a peer shared with you from your wallet, but you can't delete Attributes you shared with a peer from their wallet.
+Wanting to do so, you need to send a Request to the peer, asking them to delete the respective Attribute.
+Note that this doesn't automatically deletes their Attribute, since the peer may have a valid reason to still keep it for a certain amount of time.
+
+## Request the deletion of own Attributes from peer
+
+Wanting to delete a peer shared Attribute owned by you from the peer technically describes the endeavor of reverting sharing the Attribute with them.
+To this end, a [Request]({% link _docs_integrate/data-model-overview.md %}#request) must be used with a [DeleteAttributeRequestItem]({% link _docs_integrate/data-model-overview.md %}#deleteattributerequestitem).
+As a parameter, the `attributeId` of the peer shared Attribute you would like the peer to delete must be provided.
+Note, that the peer shared Attribute at the peer's side has the same `id` like the own shared Attribute at your side.
+A possible Request for deleting a peer shared Attribute from a peer could look as follows:
+
+```json
+{
+  "@type": "Request",
+  "items": [
+    {
+      "@type": "DeleteAttributeRequestItem",
+      "mustBeAccepted": true,
+      "attributeId": "<ID of peer shared LocalAttribute>"
```

Comment:
'peer shared Attribute' instead of 'peer shared LocalAttribute' as in the JSON payload below.